src/bin/sage-env

```diff
@@ -643,3 +644,7 @@ fi
 # our doctests predictable (which was the status quo with earlier
 # versions of debugpy).
 export PYDEVD_DISABLE_FILE_VALIDATION=1
+
+# Finally, set SAGE_ENV_SOURCED as evidence that this script has been
+# run successfully.
+export SAGE_ENV_SOURCED="6:$SAGE_LOCAL:$SAGE_VENV:$SAGE_SRC"
```

Comment:
what is that `"6:` doing?

Reply:
That's explained earlier in the file and I didn't repeat the comment:
```
# "6" indicates the version of the format of the value of SAGE_ENV_VERSION.
```

Reply:
ah, OK